### ISSUE TITLE
Ability to abandon/cancel disk import

### ISSUE THREAD

**Issue by askfongjojo** [oxidecomputer/omicron#2987]
The action is meant to allow user to remove a disk that can't finish blocks import for any reasons. What happens behind the scenes may be similar to a combination of `bulk-write-stop` and `finalize` actions. This may be more a need for the CLI because it obscures the need to manage the granular actions/states, to the extent that  when the import fails to complete, user is left with a disk that is taking up space but cannot be deleted.

I've putting this as a post-MVP requirement and will document the disk state diagrams like the following in the user guide and augment them with instructions for handling disk deletion.
![disk-create-fsm](https://user-images.githubusercontent.com/10874909/235760033-37b42eea-b2fe-4ae9-bae5-b47be1fe5020.png)
![disk-use-fsm](https://user-images.githubusercontent.com/10874909/235760034-4c047fae-7a92-4beb-af94-29581c7736dd.png)

**Comment by askfongjojo** [comment]
Customer ran into this after hitting `Ctrl-C` in the middle of importing an image using the CLI. The user (fleet admin) managed to find the two API calls necessary to unstick the disk but suspected that his users would not know what to do in this situation.

**Comment by ahl** [comment]
@askfongjojo would this be ameliorated by having a signal handler in the CLI that tries to clean up? Obviously this wouldn't be perfect (e.g. `SIGKILL` or power loss), but might it handle 95% of cases?

**Comment by david-crespo** [comment]
We do that in the console.

https://github.com/oxidecomputer/console/blob/73263a4e805cdf5757d5441b962a453bc1531c87/app/forms/image-upload.tsx#L303-L331

**Comment by askfongjojo** [comment]
> would this be ameliorated by having a signal handler in the CLI that tries to clean up? Obviously this wouldn't be perfect (e.g. `SIGKILL` or power loss), but might it handle 95% of cases?

Yes, having CLI replicate what the console does definitely covers >95% of the cases.

**Comment by askfongjojo** [comment]
De-prioritizing this (by unsetting the milestone) in favor of https://github.com/oxidecomputer/oxide.rs/issues/651.
